test/hotspot/jtreg/runtime/appcds/CDSandJFR.java

```diff
@@ -51,6 +51,8 @@ public class CDSandJFR {
         "Hello",
         "GetFlightRecorder",
         "GetFlightRecorder$TestEvent",
+        "jdk/jfr/events/FileReadIOStatisticsEvent",
```

Comment:
NIT: can you move these "jdk/jfr/..." tests closer to the ones above?

Reply:
I moved the tests closer to the other fileEvent tests. Thanks for the suggestion.